whois-api/src/main/java/net/ripe/db/whois/api/elasticsearch/ElasticIndexService.java

```diff
@@ -92,7 +92,7 @@ public boolean isEnabled() {
         return true;
     }
 
-    protected void addEntry(final RpslObject rpslObject) throws IOException {
+    protected void updateIfExistCreateIfNot(final RpslObject rpslObject) throws IOException {
```

Comment:
rename to CreateOrUpdateEntry? Although  addEntry  is also correct as operation for both update and create is Update only

Reply:
If you check the output of the index is has two responses
CREATE or UPDATE
so I think that addEntry is not a good name, we are mixing things even if we are calling same service.
but I will do the change

Reply:
No worries, I am ok with This name as well. I meant we are updating the entries via serial entry table and it has only two operation update and delete so update is both for create and update so we do not differentiate between them.